### ISSUE TITLE
Interactions between hubs

### ISSUE THREAD

**Issue by tukkek** [tukkek/minetest_rpg#2]
In order to promote exploration, quests and items should create interactions between hubs. Examples:
* A magic compass could point to the nearest hub in a given radius - so that players can go explore and still find hubs reliably
* A new NPC could give you packages to deliver to the nearest hub (creates `parcel` item with a given coordinate objective and grants experience once it's use nearby enough)

**Comment by tacotexmex** [comment]
Sounds nice. Check out the ”ccompass” mod by bell007 (textures by me 😁), it’s what you’re looking for.

**Comment by tukkek** [comment]
Looks like a perfect match, thanks! If you make textures would you be interested in making a logo, background and banner for the game version of MineRPG? It would help a whole lot! (I decided to keep mod and game separated by the way, based on what you said earlier on that other issue)

**Comment by tacotexmex** [comment]
Perhaps, but I’m currently very short of computer time. Create some makeshift graphics first and then later I might be able to assist. A game concept would help in that regard.

**Comment by tukkek** [comment]
Yeah, I'm currently just using a free-to-use `icon.png`, which is all that is really required. Do you mind if I create a new issue for this and tag you in it? Sadly I have zero artistic skills to do it myself, so it would be a great help :P
